src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryService.java

```diff
@@ -23,7 +26,7 @@ public CftQueryService(TaskResourceRepository taskResourceRepository) {
         this.taskResourceRepository = taskResourceRepository;
     }
 
-    public List<TaskResource> getAllTasks(
+    public GetTasksResponse<Task> getAllTasks(
```

Comment:
wouldn't be better to keep List<TaskResource> at this level and not use the GetTasksResponse which is the upper level?

Reply:
following the standards , making them consistent across the apis.

Reply:
Which standards are you referring to? i think class dependency should be downward not upward, it should not depend on class which are in the controllers package. We ask other devs if you want.

Reply:
I got it. will change.

Reply:
The reason being the totalNumber of records cannot be added separately in controller. For MVP , to get total number or records extra call made to get total number of records but in jpa case its overkill.